.github/workflows/publish.yml

```diff
@@ -126,13 +126,15 @@ jobs:
         rm _iconCloud/manifest.json.bk
 
     ## Publish svg icons
-    - name: Run svg publish
-      run: npm publish
+    - name: Build and publish SVGs to npm
+      run: |
+        npm install
+        npm config set //registry.npmjs.org:_authToken ${NODE_AUTH_TOKEN}
+        npm run build
+        npm publish --ignore-scripts
```

Comment:
I suspected the order might be off, like `with` needed to be before `run`. Curious here why the `ignore-scripts` was necessary. Seems like the `prepublishOnly` script is more encapsulated.